aries-site/src/examples/templates/filtering/FilteringCards/FilteringCards.js

```diff
@@ -63,73 +62,83 @@ export const FilteringCards = ({ containerRef }) => {
       <Heading level={2} margin="none">
         Users
       </Heading>
-      <FiltersProvider>
-        <Box gap="medium">
-          <FilterControls
-            data={users}
-            filters={filtersConfig}
-            layerProps={layerProps}
-            searchFilter={{ placeholder: 'Search' }}
-          />
+      <Box fill>
+        <Data
+          data={users}
+          updateOn="change"
+          height={{ min: 'medium', max: '100%' }}
+        >
+          <Toolbar>
+            <DataSearch />
+            <DataFilters layer>
```

Comment:
I don't see the submit buttons in the layer. Thinking we should change this `updateOn` to submit